lib/ansible/modules/network/nsupdate.py

```diff
@@ -40,7 +40,7 @@
   - dnspython
 author: "Loic Blot (@nerzhul)"
 options:
-    state:
+    state:e
```

Comment:
@nerzhul sorry about that ; I somehow missed it checking the commit.  I have rebased off the most recent devel and at the same time dropped this from the changes.